assignments/objects.js

```diff
@@ -49,16 +86,25 @@ const example = {
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-const parent = {}
+const parent = {
+  "name": "Susan",
+  "age": 70,
+  child: {"name": "George", 
+  "age": 50,
+  grandchild: {"name": "Sam", "age": 30},
+  },
+  speak: function() {
+    console.log(`Hi! My name is ${this.name}`);
+  }
+}
```

Comment:
Great job attempting the stretch problem. Notice that it's asking for the grandchild to be nested inside the child (i.e. the Sam object should be inside the George object) and that a speak function be available/added to each.